newsrevenuehub/rev-engine
# Dev 1818 primary nav links in page editor

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
Primary nav should not show in edit page. This was a regression after merge of flags feature ( that also included handling of 404 )

#### What's this PR do?
The primary nav is not visible when user edits a donations page

#### Why are we doing this? How does it help us?
For better user experience.

#### How should this be manually tested?
Try to edit any donation page. The primary-nav on left should not be visible when page is being edited.

#### Have automated unit tests been added?
No. 

#### How should this change be communicated to end users?
n/a

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-1818

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

#### Are there next steps? If so, what? Have tickets been opened for them?
